tests/dummy/app/routes/demos/grid.js

```diff
@@ -0,0 +1,52 @@
+import Ember from 'ember';
+
+const LETTERS = [
+    'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
+    'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'y', 'z'
+];
+
+function capitalize( string ) {
```

Comment:
I know this is just a test but http://emberjs.com/api/classes/Ember.String.html#method_capitalize did not serve your needs?